npm/npm.go

```diff
@@ -313,3 +315,16 @@ func NewNetworkPolicyManager(clientset *kubernetes.Clientset, informerFactory in
 
 	return npMgr
 }
+
+// reconcileChains checks for ordering of AZURE-NPM chain in FORWARD chain periodically.
+func (npMgr *NetworkPolicyManager) reconcileChains() {
+	iptMgr := iptm.NewIptablesManager()
+	var err error
+	for {
+		time.Sleep(reconcileChainTimeInMinutes * time.Minute)
```

Comment:
Probably better to use `time.After` in a `for select` if you ever intend to do handle termination gracefully, since that gives you a hook for a context.